data/morphology/grab_unimorph_data.py

```diff
@@ -1,52 +1,69 @@
 #!/usr/bin/env python
 """Downloads UniMorph morphological paradigms data."""
-
 import json
 import logging
 import os
-import requests
 import time
-
 from typing import Dict
 
 
+""" import requests in separate code block because of third party status """
+import requests
+
+
+start_time = time.time()
```

Comment:
The wall-clock time stuff doesn't really need to live in here. If the user cares they can just use `time` from the command-line.